README.md

```diff
@@ -1,2 +1,56 @@
 # sustainable-society
 Collection of documents to use when needing an advice regarding sustainability 
+
+## Food
+-  Buy local and seasonal food, no need to buy summer food in winter, or if you do have cravings for it buy (or do it yourself) pickled or in naother way preserved version
+
+- Avoid supermarkets for most of the things, buy there only the things that are not available locally
+
+- Eat less meat, it's more sustainable and healthier
+
+## Clothing
+-  Prefer clothing brands that promote unisex, one-size-fits-all models, that are against fast fashion and use quality materials that are durable, so you can wear e.g. one T-shirt for a long time, or even swap with friends or let your kids inherit it. Example [Older brother]( https://olderbrother.us) or [Patagonia](https://www.patagonia.com/home/)
+
+- Use [eco-egg](https://www.ecoegg.com) instead of detergents to wash the clothes
```

Comment:
It won't completely replace detergents, but ecoegg is a good replacement for more than 50% of your wash cycles. Otherwise, look for detergents that are natural and come in a cardbox.
If you use softener, use it less and also find one that is not toxic for the environment.